psi4/src/psi4/libmints/mintshelper.cc

```diff
@@ -785,11 +774,11 @@ SharedMatrix MintsHelper::so_dkh(int dkh_order) {
     return dkh;
 }
 
-SharedMatrix MintsHelper::ao_helper(const std::string &label, std::shared_ptr<TwoBodyAOInt> ints) {
-    std::shared_ptr<BasisSet> bs1 = ints->basis1();
-    std::shared_ptr<BasisSet> bs2 = ints->basis2();
-    std::shared_ptr<BasisSet> bs3 = ints->basis3();
-    std::shared_ptr<BasisSet> bs4 = ints->basis4();
+SharedMatrix MintsHelper::ao_helper(const std::string &label, std::vector<std::shared_ptr<TwoBodyAOInt>> ints) {
```

Comment:
why the change?

Reply:
I wanted to be consistent btwn the `one_body_ao_computer` and the `ao_helper`. The OneBody case uses `std::vector<std::shared_ptr<OneBodyAOInt>> ints` so the TwoBody case does as well.